packages/polling-controller/src/PollingController.test.ts

```diff
@@ -0,0 +1,214 @@
+import { ControllerMessenger } from '@metamask/base-controller';
+
+import type { PollingCompleteType } from './PollingController';
+import PollingController from './PollingController';
+
+describe('PollingController', () => {
+  let executePollMock: PollingController<any, any, any>['executePoll'];
+
+  beforeEach(() => {
+    executePollMock = jest.fn().mockImplementation(async () => {
+      return true;
+    });
+  });
+
+  describe('start', () => {
+    it('should start polling if not polling', () => {
+      jest.useFakeTimers();
+
+      class MyGasFeeController extends PollingController<any, any, any> {
+        executePoll = executePollMock;
+      }
+      const mockMessenger = new ControllerMessenger<any, any>();
+
+      const controller = new MyGasFeeController({
+        messenger: mockMessenger,
+        metadata: {},
+        name: 'PollingController',
+        state: { foo: 'bar' },
+      });
+      controller.start('mainnet');
+      jest.advanceTimersByTime(1500);
```

Comment:
Nit: When I first read this, it wasn't clear why we were waiting 1500 milliseconds specifically. I see that it's 1.5 x the interval though. Maybe we could update this to use a constant that makes that more clear?